src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -432,6 +432,8 @@ public static string CertMustHavePrivateKey(string certificateName)
 
         public const string RequestFailureErrorMessagePii = "=== Token Acquisition ({0}) failed:\n\tAuthority: {1}\n\tClientId: {2}.";
 
+        public const string UnableToParseAuthenticationHeader = "Msal is unable to parse the authentication reader returned from the endpoint";

```

Comment:
Maybe more details here would be helpful for possible reasons why this may happen, like wwwAuth and auth-info headers missing, error parsing, etc?